indra/llcommon/llthread.cpp

```diff
@@ -269,6 +269,7 @@ void LLThread::shutdown()
             mStatus = STOPPED;
             return;
         }
+        delete mThreadp;
```

Comment:
I'd rather see `mThreadp` be a `std::unique_ptr<std::thread>`, with a `reset()` call here and in `start()`.

Reply:
Thank you, that's probably outside of a 'contribution' work, I will do an issue for that later.